app/controllers/shelters_controller.rb

```diff
@@ -48,12 +48,8 @@ def shelter_params
   end
 
   def remove_favorited_pets(shelter)
-    pets = shelter.pets.map { |pet| "#{pet.id}" }
-    pets.each do | pet_id |
-      if favorite.contents.include? pet_id
-        favorite.remove_pet(pet_id)
-      end
-    end
+    pets = shelter.pets.map(&:id)
+    pets.each { |pet_id| favorite.remove_pet(pet_id.to_s) }
```

Comment:
I didn't realize that our `remove_pet` method was already converting the ids into strings, so the `.to_s` here is redundant.